scalyr_agent/builtin_monitors/journald_monitor.py

```diff
@@ -557,16 +557,12 @@ def format_msg(
     ):
         string_buffer = six.StringIO()
 
-        string_buffer.write(
-            "%s %s" % (metric_name, scalyr_util.json_encode(metric_value))
-        )
+        string_buffer.write("%s %s" % (metric_name, metric_value))
 
         if extra_fields is not None:
             for field_name in extra_fields:
```

Comment:
I know this isn't your doing, but we probably should have a deterministic ordering on the fields.. so the keys or something.

Reply:
Yeah, that was also the reason for one of the tests checking individual parts of the message rather than the whole thing, we can just sort them I guess.